```
======================================================================
ERROR: custom.enikshay.integrations.ninetyninedots.tests.test_repeaters:TestRegisterPatientPayloadGenerator.test_get_payload
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/custom/enikshay/integrations/ninetyninedots/tests/test_repeaters.py", line 298, in test_get_payload
    self._assert_payload_equal(cases, expected_numbers)
  File "/mnt/commcare-hq-ro/custom/enikshay/integrations/ninetyninedots/tests/test_repeaters.py", line 274, in _assert_payload_equal
    episode_case_properties.get(TREATMENT_SUPPORTER_LAST_NAME, ''),
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-6: ordinal not in range(128)
-------------------- >> begin captured logging << --------------------
```

--- a/custom/enikshay/integrations/ninetyninedots/tests/test_repeaters.py
+++ b/custom/enikshay/integrations/ninetyninedots/tests/test_repeaters.py
@@ -269,7 +269,7 @@ class TestPayloadGeneratorBase(ENikshayCaseStructureMixin, ENikshayLocationStruc
             "phone_numbers": expected_numbers,
             "merm_id": person_case_properties.get(MERM_ID, None),
             "treatment_start_date": episode_case_properties.get(TREATMENT_START_DATE, None),
-            "treatment_supporter_name": "{} {}".format(
+            "treatment_supporter_name": u"{} {}".format(
                 episode_case_properties.get(TREATMENT_SUPPORTER_FIRST_NAME, ''),
                 episode_case_properties.get(TREATMENT_SUPPORTER_LAST_NAME, ''),
             ),

